wiki/People/Users_with_unique_titles/id.md

```diff
@@ -1,18 +1,14 @@
----
-outdated: true
----
+# Daftar pengguna dengan gelar khusus
 
-# Pengguna dengan gelar unik
+Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [gugus pengguna (*user group*)](/wiki/People/The_Team#user-groups). Meskipun demikian, terdapat beberapa pengguna yang secara khusus dianugerahi gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang telah mereka torehkan.
```

Comment:
```suggestion
Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [gugus pengguna (*user group*)](/wiki/People/The_Team#gugus-pengguna). Meskipun demikian, terdapat beberapa pengguna yang secara khusus dianugerahi gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang telah mereka torehkan.
```

Reply:
the id page is changing and should be redirected 
![image](https://user-images.githubusercontent.com/39128490/120093143-76cbc480-c142-11eb-9fa7-6e007af628b2.png)

Reply:
missed this somehow whoops, tyty